viewer/packages/360-images/src/cache/Image360LoadingCache.ts

```diff
@@ -61,4 +88,21 @@ export class Image360LoadingCache {
     pull(this._loaded360Images, entityToPurge);
     entityToPurge.unload360Image();
   }
+
+  private abortLastRecentlyReqestedEntity() {
+    const entityToAbort = Array.from(this._inFlightEntities.keys())[0];
```

Comment:
Does this really ensure that it's the least recently requested entity that's removed?

Reply:
Changed from map to array 🗺️